t/lib/OpenQA/Test/Utils.pm

```diff
@@ -215,7 +215,9 @@ sub _setup_sub_process {
     $0 = $process_name;
     note "PID of $process_name: $$";
     $SIG{__WARN__} = sub {
-        log_error "Stopping $process_name process because a Perl warning occurred: @_";
+        my $warning = "@_";
+        log_error "Stopping $process_name process because a Perl warning occurred: $warning"
+          unless $warning =~ qr/DEPRECATED/i;
```

Comment:
as discussed please create a ticket for that. Then I am completely fine to skip that. Good.

Reply:
This doesn't seem to work:

```perl
#   Failed test 'no (unexpected) warnings (via done_testing)'
#   at t/39-scheduled_products-table.t line 88.
# Got the following unexpected warnings:
#   1: Mojolicious::Routes::Route::route is DEPRECATED in favor of Mojolicious::Routes::Route::any at /usr/lib/perl5/vendor_perl/5.26.1/Mojolicious/Plugin/AssetPack.pm line 14.
#   2: Mojolicious::Routes::Route::via is DEPRECATED in favor of Mojolicious::Routes::Route::methods at /usr/lib/perl5/vendor_perl/5.26.1/Mojolicious/Plugin/AssetPack.pm line 14.
# Looks like you failed 1 test of 3.
```